tools/CouchDB_Import_Demographics.php

```diff
@@ -16,6 +16,10 @@ class CouchDBDemographicsImporter {
             'Description' => 'Date of Birth',
             'Type' => 'varchar(255)'
         ),
+        'DoD' => array(
+            'Description' => 'Date of Death',
+            'Type' => 'varchar(255)'
```

Comment:
This has the Date type in your patch, not varchar(255). It feels like these should match.

Reply:
I was following what was done for Date of Birth (above), should I change it ?

Reply:
That makes sense. @ridz1208 can you comment here? I don't know enough about CouchDB to know what's correct here.

Reply:
yes date is better

you can change the dob one too